test/integration/backend.cc

```diff
@@ -0,0 +1,90 @@
+#include <celerity.h>
+
+#define ASSERT(...)                                                                                                                                            \
+	if(!(__VA_ARGS__)) {                                                                                                                                       \
+		CELERITY_ERROR("ASSERT({}) on line {} failed\n", #__VA_ARGS__, __LINE__);                                                                              \
+		std::_Exit(EXIT_FAILURE);                                                                                                                              \
```

Comment:
This should be an `abort()` to trigger a core dump / debugger trap.

Reply:
Done!